docs/sources/release-notes/v2.5.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Grafana Mimir version 2.5 release notes"
+menuTitle: "V2.5 release notes"
+description: "Release notes for Grafana Mimir version 2.5"
+weight: 300
+---
+
+# Grafana Mimir version 2.5 release notes
+
+Grafana Labs is excited to announce version 2.5 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- Alertmanager discord support
+- TLS minimum version and cipher suites are now configurable for HTTP and gRPC servers
+- Various improvements to lower memory usage of store-gateway and ingester
+- New dashboards: "Mimir / Overview resources" and Mimir / Overview networking"
+- Support for read-write deployment mode in jsonnet, this allows users to run Mimir as only three types of services which are responsible for read, write and backend operations
+
+## Bug fixes
+
+* Flusher: Add `Overrides` as a dependency to prevent panics when starting with `-target=flusher`. [PR 3151](https://github.com/grafana/mimir/pull/3151)
+* Query-frontend: properly close gRPC streams to the query-scheduler to stop memory and goroutines leak. [PR 3302](https://github.com/grafana/mimir/pull/3302)
+* Ruler: persist evaluation delay configured in the rulegroup. [PR 3392](https://github.com/grafana/mimir/pull/3392)
+* Ring status pages: show 100% ownership as "100%", not "1e+02%". [PR 3435](https://github.com/grafana/mimir/pull/3435)
```

Comment:
This is really a nit. I wouldn't show it.

Reply:
removed 644e3b0c5